specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2018-02-14-preview/keyvault.json

```diff
@@ -51,13 +51,13 @@
           }
         ],
         "responses": {
-          "201": {
+          "200": {
```

Comment:
This seems to have nothing to do with setting `modelAsString=false` and is a breaking change.